.gitignore

```diff
@@ -3,6 +3,6 @@ work/
 data/
 results/
 .DS_Store
-tests/
 testing/
+testing*
```

Comment:
Just for me to know, what is the reason for this change in the template?

Reply:
No idea!